docs/source/tutorials/exploration.rst

```diff
@@ -304,12 +304,12 @@ the population as a whole.
     alive    100000
     Name: count, dtype: int64
     count    100000.000000
-    mean          0.500602
```

Comment:
I assume this is perfectly expected that some things would chance since we're actually mucking around with the randomness hash?